### PR TITLE
Create FUNDING.yml

### PR DESCRIPTION
**Pull request by MichaelAndrew22**
funding page experiment. @Cxrdary @MichaelAndrew22 @crazyne55 ### test

### CODE REVIEW

**Comment by MichaelAndrew22** [comment]
testing

**Comment by crazyne55** [comment]
I think the format for including multiple sponsors is [sponsor1, sponsor2, ...] instead of just sponsor1, sponsor2, ..., I could be wrong though.

File: .github/FUNDING.yml
Comment (crazyne55) [inline .github/FUNDING.yml:1]: Like I mentioned in the conversation tab, I'm pretty sure the format for arrays/lists in yml is [value1,value2,value3,...], other than that, I'd say this is pretty good

**Review by crazyne55 — APPROVED** [review]
not gonna lie, this review system is a tad confusing, but I think I am getting the hang of it.

we could give the current state of the funding.yml a try, but if it doesn't work right, then we should change it to use the list format I suggested

**Review by Cxrdary — COMMENTED** [review]
seen

**Comment by Cxrdary** [comment]
No funding need.

**Pull request closed without merging**